content/projects/gsoc/2023/projects/add-probes-to-plugin-health-score.adoc

```diff
@@ -9,36 +9,57 @@ tags:
 - plugins
 status: "Selected"
 sig: platform
-//FIXME: create bio for Jagruti Tiwari in content/_data/authors/
-student: "null_gsoc_contributor"
+student: "jagruti"
 mentors:
 - "alecharp"
 - "dheerajodha"
 - "mostafaashraf"
 links:
     idea: /projects/gsoc/2023/project-ideas/add-probes-to-plugin-health-score
-//   gitter: "some gitter channel"
-//   draft: link to the draft/final proposal
-//   meeting: link to internal section of this page with the meeting details and notes
+   gitter: https://app.element.io/\#/room/\#jenkinsci_GSoC-Plugin_Health_Score:gitter.im
```

Comment:
Items must be aligned otherwise the generation crashes
```suggestion
    gitter: https://app.element.io/\#/room/\#jenkinsci_GSoC-Plugin_Health_Score:gitter.im
```